charts/__tests__/DataTable.test.tsx

```diff
@@ -6,23 +6,22 @@ import { ChartConfig, ChartConfigProps } from "../ChartConfig"
 import { extend } from "charts/Util"
 import * as fixtures from "test/fixtures"
 
-const chartFixture = fixtures.readChart(677)
-const variableFixture = fixtures.readVariable(104402)
-
-describe(DataTable, () => {
+function setupChart(id: number, varId: number) {
     const props = new ChartConfigProps()
-    let chart: ChartConfig
+    extend(props, fixtures.readChart(id))
```

Comment:
I made it possible to pass initializing props to the `ChartConfigProps` constructor, it might be more convenient to use that (you also get typechecking with it)

Reply:
Nice, much cleaner